alumnium/alumni.py

```diff
@@ -75,28 +73,25 @@ def do(self, goal: str):
         for step in steps:
             self.actor_agent.invoke(goal, step)
 
-    def check(self, statement: str, vision: bool = False) -> Verification:
+    def check(self, statement: str, vision: bool = False) -> str:
         """
         Checks a given statement using the verifier.
 
         Args:
             statement: The statement to be checked.
             vision: A flag indicating whether to use a vision-based verification via a screenshot. Defaults to False.
+            retries: The number of retries to check the statement. Defaults to the value set in the LoadingDetectorAgent.
```

Comment:
@p0deje hm, there is no `LoadingDetectorAgent` anymore, right?

Reply:
I'll remove